### DISCUSSION TITLE
Rotation of axes in seaborn.objects plots

### DISCUSSION THREAD

**Discussion by antunderwood** [mwaskom/seaborn#3090] [Q&A]
There are some features of the plots such as rotation of axes that does not readily seem available via the `.scale()` interface. Please can you advise and add to the Docs. 

Thanks

**Comment by mwaskom** [comment]
What do you mean by “rotation of axes”?

**Comment by antunderwood** [comment]
I would like to rotate long x tick labels by e.g 45 or 90 degrees so they do not overlap with one another. Thanks
